pkgs/tools/misc/gsmartcontrol/default.nix

```diff
@@ -1,18 +1,22 @@
 { fetchurl, lib, stdenv, smartmontools, autoreconfHook, gettext, gtkmm3, pkg-config, wrapGAppsHook, pcre-cpp, gnome }:
 
 stdenv.mkDerivation rec {
-  version="1.1.3";
+  version="1.1.4";
   pname = "gsmartcontrol";
 
   src = fetchurl {
-    url = "mirror://sourceforge/gsmartcontrol/gsmartcontrol-${version}.tar.bz2";
-    sha256 = "1a8j7dkml9zvgpk83xcdajfz7g6mmpmm5k86dl5sjc24zb7n4kxn";
+    url = "https://github.com/ashaduri/gsmartcontrol/releases/download/v${version}/gsmartcontrol-${version}.tar.bz2";
```

Comment:
Why are we using that tarball instead of the git archive?

Reply:
This package uses autoreconfHook and hence requires *.in template files to run. tar.bz2 archive [contains](https://github.com/ashaduri/gsmartcontrol/releases/tag/v1.1.4) these files, while tar.gz doesn't contain them. I guess we could rewrite our package to use plain makefiles, but it would require additional effort. Since I'm not a maintainer of this package and just want to submit fix I've made for myself upstream, I could add TODO commentary to encourage future nixers to try and rewrite package definition next time.

Reply:
Yes, you're correct. For some reason tar.gz on the release page doesn't contain autoconf files, but tar.gz2 archive has them inside.

Reply:
Urg, ugly, but oh well.